Services/Style/classes/class.ilStyleDefinition.php

```diff
@@ -437,6 +441,28 @@ public static function getCurrentMasterStyle()
 		return $cs;
 	}
 
+	public static function getCurrentSkinVersion()
+	{
+		/** @var ilStyleDefinition $styleDefinition */
+        global $styleDefinition;
```

Comment:
The indentation seems odd, which makes it really hard to review.

Looks like you mixed up spaces with tabs? See also below.

Reply:
I checked the indentation and will change it. My editor automatically replaces tabs with 4 spaces, because this is recommended in most other projects I worked on. I have to change this configuration for ILIAS.